include/crocoddyl/multibody/fwd.hpp

```diff
@@ -472,7 +472,7 @@ DEPRECATED("Use ResidualDataContactWrenchCone", typedef CostDataResidualTpl<doub
 typedef ImpulseModelAbstractTpl<double> ImpulseModelAbstract;
 typedef ImpulseDataAbstractTpl<double> ImpulseDataAbstract;
 
-enum ContactType { Contact2D, Contact3D, Contact6D, ContactUndefined };
+enum ContactType { Contact1D, Contact2D, Contact3D, Contact6D, ContactUndefined };
```

Comment:
Here ContactUndefined should be the first element of the enum for debugging perspectives.

Reply:
@skleff1994 -- could you please handle it?

Reply:
Thank you for reminding me!  Yes I will handle it.